From ed02656b1a89912919b011c513079359ba739e4b Mon Sep 17 00:00:00 2001
From: Dmitry Baryshkov <dmitry.baryshkov@oss.qualcomm.com>
Date: Wed, 20 Aug 2025 00:53:22 +0300
Subject: [PATCH 7/8] kgsl: fix undefined platform_bus_type error

Include platform_device.h in order to fix the undeclared
platform_bus_type compiler error.

Signed-off-by: Dmitry Baryshkov <dmitry.baryshkov@oss.qualcomm.com>
Upstream-Status: Submitted [https://github.com/qualcomm-linux/kgsl/pull/9]
---
 kgsl_gmu_core.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kgsl_gmu_core.h b/kgsl_gmu_core.h
index d6f226905b2a..d3241991ae8c 100644
--- a/kgsl_gmu_core.h
+++ b/kgsl_gmu_core.h
@@ -531,6 +531,7 @@ static inline struct iommu_domain *gmu_core_iommu_domain_alloc(struct device *de
 	return iommu_paging_domain_alloc(dev);
 }
 #else
+#include <linux/platform_device.h>
 static struct iommu_domain *gmu_core_iommu_domain_alloc(struct device *dev)
 {
 	return iommu_domain_alloc(&platform_bus_type);
-- 
2.47.2

